tests/test_fft.py

```diff
@@ -2,51 +2,75 @@
 import numpy as np
 
 from aspire import config
-from aspire.utils.numeric import fft_class
+from aspire.utils.numeric import fft_object
 
 fft_backends = ['scipy', 'pyfftw']
-numeric_classes = [np, np]
+numeric_modules = [np, np]
```

Comment:
Ideally, this should be done in the same way as for FFT backends (with an object instantiation), but that should go in an issue.

Reply:
Created new issue #317.